rest/kadai-rest-spring/src/main/java/io/kadai/task/rest/TaskController.java

```diff
@@ -531,6 +535,30 @@ public ResponseEntity<TaskRepresentationModel> updateTask(
     return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(task));
   }
 
+  @PatchMapping(path = RestEndpoints.URL_TASKS_BULK_UPDATE)
+  @Transactional(rollbackFor = Exception.class)
+  public ResponseEntity<BulkOperationResultsRepresentationModel> bulkUpdateTasks(
+      @RequestBody TaskBulkUpdateRepresentationModel requestModel) {
+
+    if (requestModel.getTaskIds() == null || requestModel.getTaskIds().isEmpty()) {
+      throw new InvalidArgumentException("taskIds must not be empty");
+    }
+    if (requestModel.getFieldsToUpdate() == null) {
```

Comment:
```suggestion
    if all fields of TaskPatch are null {
```

All fields null could happen.
If the object itself is null, it's highly likely to be a client error (400).